juniper_official/Linecard/aft-server-entry-stats.rule

```diff
@@ -0,0 +1,41 @@
+healthbot {
+    topic external {
+        rule aft_server_entry_stats {
+            synopsis "Aft Server Sandbox Entry Stats";
+            description "Aft Server Sandbox Entry Stats";
+            sensor aft_sb_entry_stats {
+                iAgent {
+                    file aft-sandbox-stats.yml;
+                    table SandboxStatsTable;
+                    frequency 180s;
+                }
+            }
+            field aft-entry {
+                sensor aft_sb_entry_stats {
+                    path aft-entry;
+                }
+            }
+            field entry-add-fail {
+                sensor aft_sb_entry_stats {
+                    path entry-add-fail;
+                }
+            }
+            field entry-create-fail {
+                sensor aft_sb_entry_stats {
+                    path entry-create-fail;
+                }
+            }
+            field entry-out-of-order {
+                sensor aft_sb_entry_stats {
+                    path entry-out-of-order;
+                }
+            }
+            field fpc-name {
+                sensor aft_sb_entry_stats {
+                    path target;
+                }
+                type string;
+            }
+        }
+    }
```

Comment:
Rule properties are mandatory. Please include same.

Reply:
There are no other properties for this rule. I will discuss more with you.